sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -0,0 +1,619 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <assert.h>
+#include <limits.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/math.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_aes.h"
+#include "sw/device/lib/dif/dif_alert_handler.h"
+#include "sw/device/lib/dif/dif_clkmgr.h"
+#include "sw/device/lib/dif/dif_flash_ctrl.h"
+#include "sw/device/lib/dif/dif_hmac.h"
+#include "sw/device/lib/dif/dif_kmac.h"
+#include "sw/device/lib/dif/dif_otbn.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/dif/dif_rv_core_ibex.h"
+#include "sw/device/lib/dif/dif_rv_plic.h"
+#include "sw/device/lib/dif/dif_spi_host.h"
+#include "sw/device/lib/dif/dif_usbdev.h"
+#include "sw/device/lib/runtime/irq.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/alert_handler_testutils.h"
+#include "sw/device/lib/testing/flash_ctrl_testutils.h"
+#include "sw/device/lib/testing/rv_plic_testutils.h"
+#include "sw/device/lib/testing/test_framework/FreeRTOSConfig.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "aes_regs.h"            // Generated.
+#include "alert_handler_regs.h"  // Generated.
+#include "hmac_regs.h"           // Generated.
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "i2c_regs.h"
+#include "kmac_regs.h"  // Generated.
+#include "otbn_regs.h"  // Generated.
+#include "spi_host_regs.h"
+#include "sw/device/lib/testing/autogen/isr_testutils.h"
+#include "usbdev_regs.h"
```

Comment:
The `// Generated.` line looks a little inconsistent here -- isn't `spi_host_regs.h` also generated? I only see it under `build-out` when I run `find`. I think it's a little inconsistent throughout the codebase, so I'd personally be fine with either adding it to all autogenerated headers (probably `i2c_regs.h`, `spi_host_regs.h`, and `usbdev_regs.h`) or simply removing it from all these includes.